app.js

```diff
@@ -0,0 +1,51 @@
+let library = [];
+let counter = 0;
+function getInput() {
+  const books = {};
+  books.id = counter;
+  books.title = document.getElementById('title').value;
+  books.author = document.getElementById('author').value;
+  return books;
+}
+
+function removeBook(id) {
```

Comment:
Remove book functionality does not work as expected. It removes all the books when a user tries to remove one of them if all have the same title or author. Moreover, it does not remove the same book which the user wants to remove.